### PR TITLE
Updating version to 0.2.0

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
   <groupId>com.google.auth</groupId>
   <artifactId>google-auth-library-parent</artifactId>
-  <version>0.2.0-SNAPSHOT</version>
```
Comment (anthmgoogle) [inline pom.xml]: This is the right version change. It also needs to be updated at the top of the individual sub-component pom.xml files also. We have explored this before, and there is not a known way to share this value in a single field accross the poms.

**Comment by anthmgoogle** [comment]
Need to update the other POMs also.

**Comment by garrettjonesgoogle** [comment]
I have updated the subcomponent pom.xml files too now.

**Comment by anthmgoogle** [comment]
LGTM. I don't think we needed anything else in this minor version, so merge when you are ready.

Once the version is updated, the next update before any other changes go in should be to change it to 0.3.0-SNAPSHOT.

**Pull request closed without merging**